packages/graphql/tests/tck/sort.test.ts

```diff
@@ -665,4 +664,300 @@ describe("Top-level Interface query sort", () => {
             }"
         `);
     });
+
+    describe("add @limit directive on interface", () => {
+        beforeAll(() => {
+            typeDefs = typeDefs + `extend interface MyInterface @limit(default: 13, max: 15) `;
+
+            neoSchema = new Neo4jGraphQL({
+                typeDefs,
+                experimental: true,
```

Comment:
I worry about these feature flags, that may be here forever. Should we move experimental features tests into a separate folder? (similar to deprecations)